core/domain/html_cleaner.py

```diff
@@ -273,8 +279,7 @@ def convert_to_text_angular(html_data):
         # is None and there is no need to add tabs since linebreak is
         # already present.
         elif tag.name == 'td' and tag.next_sibling:
-            if tag.string:
-                tag.string = tag.string + "\t"
+            tag.insert_after('\t')
```

Comment:
Are tab chars allowed / possible to produce in textAngular?

Reply:
My bad here - I discussed this point with you on docs but missed it here, I have fixed this now.

Also the line `tag.string = tag.string + "\t"` was the line which resulted in exception earlier, that is fixed now as well.